app/components/markdown.tsx

```diff
@@ -59,3 +59,126 @@ export const Markdown = (
     />
   );
 };
+
+const componentsInheritFonts: ComponentProps<
+  typeof ReactMarkdown
+>["components"] = {
+  h1: ({ children, style, ...props }) => (
+    <h1
+      style={{
+        ...style,
+        marginTop: 0,
+        fontSize: "inherit",
+        fontWeight: "inherit",
+      }}
+      {...props}
+    >
+      {children}
+    </h1>
+  ),
+  h2: ({ children, style, ...props }) => (
+    <h2
+      style={{
+        ...style,
+        marginTop: 0,
+        fontSize: "inherit",
+        fontWeight: "inherit",
+      }}
+      {...props}
+    >
+      {children}
+    </h2>
+  ),
+  h3: ({ children, style, ...props }) => (
+    <h3
+      style={{
+        ...style,
+        marginTop: 0,
+        fontSize: "inherit",
+        fontWeight: "inherit",
+      }}
+      {...props}
+    >
+      {children}
+    </h3>
+  ),
+  h4: ({ children, style, ...props }) => (
+    <h4
+      style={{
+        ...style,
+        marginTop: 0,
+        fontSize: "inherit",
+        fontWeight: "inherit",
+      }}
+      {...props}
+    >
+      {children}
+    </h4>
+  ),
+  h5: ({ children, style, ...props }) => (
+    <h5
+      style={{
+        ...style,
+        marginTop: 0,
+        fontSize: "inherit",
+        fontWeight: "inherit",
+      }}
+      {...props}
+    >
+      {children}
+    </h5>
+  ),
+  h6: ({ children, style, ...props }) => (
+    <h6
+      style={{
+        ...style,
+        marginTop: 0,
+        fontSize: "inherit",
+        fontWeight: "inherit",
+      }}
+      {...props}
+    >
+      {children}
+    </h6>
+  ),
+  p: ({ children, style, ...props }) => (
+    <p
+      style={{
+        ...style,
+        marginTop: 0,
+        fontSize: "inherit",
+        fontWeight: "inherit",
+      }}
+      {...props}
+    >
+      {children}
+    </p>
+  ),
+  a: ({ children, style, ...props }) => (
+    <a
+      target="_blank"
+      style={{
+        ...style,
+        marginTop: 0,
+        fontSize: "inherit",
+        fontWeight: "inherit",
+      }}
+      {...props}
+    >
+      {children}
+    </a>
+  ),
+};
```

Comment:
Nit: Perhaps a className (via styles.module.css) or a shared style object could be cleaner ?

Reply:
Nice idea, thanks 👍